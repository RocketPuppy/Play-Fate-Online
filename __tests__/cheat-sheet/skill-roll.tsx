import * as React from "react";
import * as renderer from "react-test-renderer";
import SkillRoll from "../../src/cheat-sheet/skill-roll";

test("renders all the pieces", () => {
  const render = renderer.create(<SkillRoll />);

  expect(render.toJSON()).toMatchSnapshot();
});
