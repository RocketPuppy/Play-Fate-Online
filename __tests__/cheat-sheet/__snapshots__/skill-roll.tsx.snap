// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Skill Roll
  </h2>
  <p>
    Roll four 
    <em>
      Fate dice
    </em>
     and add to 
    <em>
      skill rating
    </em>
    . Compare to opposition. For each step on the ladder greater than your opposition, you earn a 
    <em>
      shift
    </em>
    .
  </p>
</div>
`;
